docs/cli-existing-project.md

```diff
@@ -44,22 +43,7 @@ In this guide, you'll propose the deployment on the command line and then approv
   * [Node Version >=16.16.0](https://nodejs.org/en/download). (Run `node -v` to see your current version).
   * [Rust Compiler](https://www.rust-lang.org/tools/install)
 
-## 2. Install Sphinx Foundry Fork
-For now, Sphinx requires a [fork of Foundry](https://github.com/sphinx-labs/foundry) that resolves several bugs that conflict with Sphinx. We're working on getting these fixed in the official Foundry repo.
-
-If you are using macOS, you will first need to install the Xcode Command Line Tools:
-```
-xcode-select --install
```

Comment:
Is installing xcode not necessary anymore?

Reply:
No it should not be since this installation method does not require the compiler.

Reply:
Sweet